oak_enclave_rt/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "oak_enclave_rt"
```

Comment:
Also, I think we stopped using the term `runtime` because it was only applicable to the Wasm sandbox (since it runs other workloads)

Reply:
It's applicable here as it's used during the runtime of the application running in the enclave. Unfortunately the word "runtime" can have multiple meanings.